### PR TITLE
Create new groups for hive and hbase tests

### PR DESCRIPTION
**Pull request by lavjain**
We do not need to run hive and hbase tests for gpdb group unless explicitly stated

### CODE REVIEW

**Comment by lavjain** [comment]
@shivzone Removing from GPDB group does not mean that we will not be running those tests. The pipeline job explicitly needs to specify HDFS, HIVE, HBASE etc for the groups that it wants to run.

**Comment by frankgh** [comment]
I would probably keep the `gpdb` group and add the specific subgroup to the test (i.e `hbase`, `hive`, etc).

File: automation/src/test/java/org/greenplum/pxf/automation/features/hive/HiveTextTest.java [hunk 1]
```diff
@@ -93,7 +93,7 @@ void prepareData() throws Exception {
      *
      * @throws Exception if test fails to run
      */
-    //@Test(groups = "features")
+    @Test(groups = "features")
```
Comment (denalex) [inline automation/src/test/java/org/greenplum/pxf/automation/features/hive/HiveTextTest.java]: should this be "hive" now ?

**Comment by denalex** [comment]
I would drop `gpdb` group altogether as this framework can no longer work against HAWQ. If we also want to deprecate `features` group, then we need to group by tested subsystem (such as here : `hive`, `hbase`, `hdfs`, `s3`, `jdbc` etc) and then either provide group mapping in some test artifact or specify groups to run for pipelines. I have not seen any of these 2 artifacts in this PR.

File: automation/src/test/java/org/greenplum/pxf/automation/features/hive/HiveTextTest.java (hunk 1 above)
Comment (lavjain) [reply automation/src/test/java/org/greenplum/pxf/automation/features/hive/HiveTextTest.java]: No, because it does not work with hive on gpdb yet!

**Comment by lavjain** [comment]
@denalex Is your suggestion that we replace `gpdb` group with `feature` group and delete/comment out the current feature tests that do not work with `gpdb`.

**Comment by lavjain** [comment]
I will be merging this if there are no objections

**Pull request merged**